Makefile

```diff
@@ -93,18 +98,42 @@ isort-check: $(PELORUS_VENV)
 
 # Linting
 
-.PHONY: lint pylava chart-lint
-lint: pylava # TODO: not using chart-lint until we can automate installing it or at least conditionally run it
+.PHONY: lint pylava chart-lint chart-lint-optional shellcheck shellcheck-optional
+lint: pylava shellcheck chart-lint-optional shellcheck-optional
```

Comment:
The issue with infinite loop is that "make lint" calls shellcheck which calls ./scripts/pre-commit that calls "make -k lint" back.